src/components/SoonStartLayout/SoonStartContainer/SoonCourseCard/styledSoonCard.ts

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components";
+import { StyledPopularCardHeader } from "../../../PopularCoursesLayout/PopularCoursesContainer/PopularCard/styledPopularCard";
+import { colors, typographyColor } from "../../../../styledConstantsColors";
+
+export const StyledSoonCard = styled.div<{
+  $type?: string;
+}>`
+  border: 2px solid ;
+  border-color:  ${(props) => {
+      if (props.$type === "Design") return colors.brand.violet;
+      if (props.$type === "Data Science") return colors.brand.orange;
+      if (props.$type === "Programming") return colors.brand.green;
```

Comment:
I think it's supposed to be link, so user can go to the course description page.